generics/constants/api-responses.js

```diff
@@ -179,6 +179,6 @@ module.exports = {
   "SOLUTION_NOT_FOUND_OR_NOT_A_TARGETED": "Solution is not targeted to the role",
   "PROJECT_TEMPLATE_ID_NOT_FOUND": "Project templates not exists in solution",
   "SOLUTION_TARGETED_ENTITY" : "Targeted Entity Types Fetched",
-  "ENTITY_ID_OR_LOCATION_ID_NOT_FOUND" : "Entity id or location id not found" 
-
+  "ENTITY_ID_OR_LOCATION_ID_NOT_FOUND" : "Entity id or location id not found",
+  "NOT_AUTHORIZED_TO_ACCESS" :"The program Manager and Designer can only access this resource"
```

Comment:
@priyanka-TL please change message to - "You are not authorised to access this resource."